cli/src/pcluster/models/imagebuilder.py

```diff
@@ -370,6 +370,12 @@ def delete(self, force=False):
                     for snapshot_id in self.image.snapshot_ids:
                         AWSApi.instance().ec2.delete_snapshot(snapshot_id)
                 elif AWSApi.instance().cfn.stack_exists(self.image_name):
+                    if self.stack.instance_is_launched:
+                        raise ImageBuilderActionError(
+                            "ImageBuilder is provisioning instance, "
```

Comment:
Image cannot be deleted because EC2 ImageBuilder Image has a running workflow.

Reply:
done